docs/examples/traefik-ingress.md

```diff
@@ -90,129 +87,124 @@ root@k0s-host ➜ curl http://192.168.0.5
 404 page not found
 ```
 
-## Deploy and access the Traefik Dashboard
+## 3. Deploy and access the Traefik Dashboard
 
-Now that you have an available and addressable load balancer on your cluster, 
-you can quickly deploy the Traefik dashboard and access it from anywhere on your local network 
-(provided that you configured MetalLB with an addressable range).
+With an available and addressable load balancer present on your cluster, now
+you can quickly deploy the Traefik dashboard and access it from anywhere on
+your LAN (assuming that MetalLB is configured with an addressable range).
 
-Create the Traefik Dashboard [IngressRoute](https://doc.traefik.io/traefik/providers/kubernetes-crd/) 
-in a YAML file:
+1. Create the Traefik Dashboard [IngressRoute](https://doc.traefik.io/traefik/providers/kubernetes-crd/) in a YAML file:
 
-```yaml
-apiVersion: traefik.containo.us/v1alpha1
-kind: IngressRoute
-metadata:
-  name: dashboard
-spec:
-  entryPoints:
-    - web
-  routes:
-    - match: PathPrefix(`/dashboard`) || PathPrefix(`/api`)
-      kind: Rule
-      services:
-        - name: api@internal
-          kind: TraefikService
-```
+    ```yaml
+    apiVersion: traefik.containo.us/v1alpha1
+    kind: IngressRoute
+    metadata:
+      name: dashboard
+    spec:
+      entryPoints:
+        - web
+      routes:
+        - match: PathPrefix(`/dashboard`) || PathPrefix(`/api`)
+          kind: Rule
+          services:
+            - name: api@internal
+              kind: TraefikService
+    ```
 
-Next, deploy the resource:
+2. Deploy the resource:
 
-```bash
-root@k0s-host ➜ kubectl apply -f traefik-dashboard.yaml
-ingressroute.traefik.containo.us/dashboard created
-```
+    ```bash
+    root@k0s-host ➜ kubectl apply -f traefik-dashboard.yaml
+    ingressroute.traefik.containo.us/dashboard created
+    ```
 
-Once deployed, you should be able to access the dashboard using the `EXTERNAL-IP` 
-that you noted above by visiting `http://192.168.0.5` in your browser:
+    At this point you should be able to access the dashboard using the `EXTERNAL-IP` that you noted above by visiting `http://192.168.0.5` in your browser:
```

Comment:
"that you noted above" => "from the previous step" 🤔 ?

Reply:
I changed this text to add a bit of future-proofing to the content. If ever. a step is added between creating the dashboard and deploying it...? Always better not to use references in procedure that are too HARD.